<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeckEditorClass</class>
 <widget class="QMainWindow" name="DeckEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DeckEditor</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="mainTab">
       <attribute name="icon">
        <iconset resource="../deckeditor.qrc">
         <normaloff>:/DeckEditor/Resources/Icons/icon.png</normaloff>:/DeckEditor/Resources/Icons/icon.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Arcomage Deck Editor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Workspace</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>[$ArcomageRoot]/Editor/workspace/</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>Open selected</string>
                </property>
                <property name="icon">
                 <iconset resource="../deckeditor.qrc">
                  <normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</iconset>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>Select Workspace</string>
                </property>
                <property name="icon">
                 <iconset resource="../deckeditor.qrc">
                  <normaloff>:/DeckEditor/Resources/Icons/application-home.png</normaloff>:/DeckEditor/Resources/Icons/application-home.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QTreeWidget" name="treeWidget_3">
                <column>
                 <property name="text">
                  <string>Projects</string>
                 </property>
                </column>
                <item>
                 <property name="text">
                  <string>Workspace</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../deckeditor.qrc">
                   <normaloff>:/DeckEditor/Resources/Icons/folder-tree.png</normaloff>:/DeckEditor/Resources/Icons/folder-tree.png</iconset>
                 </property>
                 <item>
                  <property name="text">
                   <string>Empty</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../deckeditor.qrc">
                    <normaloff>:/DeckEditor/Resources/Icons/folder-horizontal.png</normaloff>:/DeckEditor/Resources/Icons/folder-horizontal.png</iconset>
                  </property>
                 </item>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>250</height>
              </size>
             </property>
             <property name="title">
              <string>Recent Files</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <property name="spacing">
               <number>6</number>
              </property>
              <item row="0" column="0" colspan="2">
               <widget class="QListWidget" name="listWidget">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Default.deck [87 Cards]</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../deckeditor.qrc">
                   <normaloff>:/DeckEditor/Resources/Icons/box.png</normaloff>:/DeckEditor/Resources/Icons/box.png</iconset>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Open selected</string>
                </property>
                <property name="icon">
                 <iconset resource="../deckeditor.qrc">
                  <normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>Clear</string>
                </property>
                <property name="icon">
                 <iconset resource="../deckeditor.qrc">
                  <normaloff>:/DeckEditor/Resources/Icons/clock-history.png</normaloff>:/DeckEditor/Resources/Icons/clock-history.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWebView" name="browser">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
          <property name="renderHints">
           <set>QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSubmit_your_Deck_online"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_Project_as"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionProject_Website"/>
    <addaction name="actionReport_Bug"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Arcomage_Deck_Editor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/document.png</normaloff>:/DeckEditor/Resources/Icons/document.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/folder-import.png</normaloff>:/DeckEditor/Resources/Icons/folder-import.png</iconset>
   </property>
   <property name="text">
    <string>Import existing Deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/disk-black.png</normaloff>:/DeckEditor/Resources/Icons/disk-black.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Project_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/disk.png</normaloff>:/DeckEditor/Resources/Icons/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save Project as...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/folder-export.png</normaloff>:/DeckEditor/Resources/Icons/folder-export.png</iconset>
   </property>
   <property name="text">
    <string>Export as .deck</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/cross.png</normaloff>:/DeckEditor/Resources/Icons/cross.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Arcomage_Deck_Editor">
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/information-frame.png</normaloff>:/DeckEditor/Resources/Icons/information-frame.png</iconset>
   </property>
   <property name="text">
    <string>About Arcomage Deck Editor</string>
   </property>
  </action>
  <action name="actionProject_Website">
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/globe.png</normaloff>:/DeckEditor/Resources/Icons/globe.png</iconset>
   </property>
   <property name="text">
    <string>Project Website</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/question-frame.png</normaloff>:/DeckEditor/Resources/Icons/question-frame.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/bug.png</normaloff>:/DeckEditor/Resources/Icons/bug.png</iconset>
   </property>
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/folder-horizontal.png</normaloff>:/DeckEditor/Resources/Icons/folder-horizontal.png</iconset>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/cog.png</normaloff>:/DeckEditor/Resources/Icons/cog.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/arrow-curve-180-left.png</normaloff>:/DeckEditor/Resources/Icons/arrow-curve-180-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/arrow-curve-000-left.png</normaloff>:/DeckEditor/Resources/Icons/arrow-curve-000-left.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</normaloff>:/DeckEditor/Resources/Icons/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSubmit_your_Deck_online">
   <property name="icon">
    <iconset resource="../deckeditor.qrc">
     <normaloff>:/DeckEditor/Resources/Icons/upload.png</normaloff>:/DeckEditor/Resources/Icons/upload.png</iconset>
   </property>
   <property name="text">
    <string>Submit your Deck online</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../deckeditor.qrc"/>
 </resources>
 <connections/>
</ui>
